libraries/adaptive-expressions/src/memory/simpleObjectMemory.ts

```diff
@@ -55,9 +55,9 @@ export class SimpleObjectMemory implements MemoryInterface {
             let error: string;
             const idx = parseInt(part);
             if(!isNaN(idx) && Array.isArray(curScope)) {
-                ({value, error} = ExpressionFunctions.accessIndex(curScope, idx));
+                ({value, error} = FunctionUtils.accessIndex(curScope, idx));
```

Comment:
> ( [](start = 15, length = 2)

Why do you need the extra parens?

Reply:
Yes. It is a format of Destructuring assignment in javascript.